libs/template/materialize.go

```diff
@@ -43,6 +44,10 @@ func Materialize(ctx context.Context, configFilePath, templateRoot, outputDir st
 	schemaPath := filepath.Join(templateRoot, schemaFileName)
 	helpers := loadHelpers(ctx)
 
+	if _, err := os.Stat(schemaPath); os.IsNotExist(err) {
+		return fmt.Errorf("expected to find a template schema file at %s", schemaPath)
```

Comment:
Suggestion: `not a bundle template: <the rest>`

Reply:
done